pkg/kubelet/kubelet.go

```diff
@@ -478,6 +480,7 @@ func NewMainKubelet(
 		return nil, err
 	}
 	klet.containerGC = containerGC
+	klet.containerDeletionWorker = newContainerDeletionWorker(klet.containerRuntime)
```

Comment:
Ideally, the deletion worker should be part of the docker runtime implementation right?